common/djangoapps/util/usermautic.py

```diff
@@ -0,0 +1,119 @@
+    "ET": "Ethiopia", "FJ": "Fiji", "FI": "Finland", "FR": "France",
+    "GA": "Gabon", "GM": "Gambia", "GE": "Georgia", "DE": "Germany",
+    "GH": "Ghana", "GR": "Greece", "GD": "Grenada", "GT": "Guatemala",
+    "GN": "Guinea", "GW": "Guinea Bissau", "GY": "Guyana",
+    "HT": "Haiti", "VA": "Holy See", "HN": "Honduras", "HU": "Hungary",
+    "IS": "Iceland", "IN": "India", "ID": "Indonesia", "IR": "Iran",
+    "IQ": "Iraq", "IE": "Ireland", "IL": "Israel", "IT": "Italy",
+    "JM": "Jamaica", "JP": "Japan", "JO": "Jordan", "KZ": "Kazakhstan",
+    "KE": "Kenya", "KI": "Kiribati", "KW": "Kuwait", "KG": "Kyrgyzstan",
+    "LA": "Laos", "LV": "Latvia", "LB": "Lebanon", "LS": "Lesotho",
+    "LR": "Liberia", "LY": "Libya", "LI": "Liechtenstein",
+    "LT": "Lithuania", "LU": "Luxembourg", "MG": "Madagascar",
+    "MW": "Malawi", "MY": "Malaysia", "MV": "Maldives", "ML": "Mali",
+    "MT": "Malta", "MH": "Marshall Islands", "MR": "Mauritania",
+    "MU": "Mauritius", "MX": "Mexico", "FM": "Micronesia",
+    "MD": "Moldova", "MC": "Monaco", "MN": "Mongolia",
+    "ME": "Montenegro", "MA": "Morocco", "MZ": "Mozambique",
+    "MM": "Myanmar", "NA": "Namibia", "NR": "Nauru", "NP": "Nepal",
+    "NL": "Netherlands", "NZ": "New Zealand", "NI": "Nicaragua",
+    "NE": "Niger", "NG": "Nigeria", "KP": "North Korea", "NO": "Norway",
+    "OM": "Oman", "PK": "Pakistan", "PW": "Palau", "PA": "Panama",
+    "PG": "Papua New Guinea", "PY": "Paraguay", "PE": "Peru",
+    "PH": "Philippines", "PL": "Poland", "PT": "Portugal",
+    "QA": "Qatar", "RO": "Romania", "RU": "Russia", "RW": "Rwanda",
+    "KN": "Saint Kitts and Nevis", "LC": "Saint Lucia", "WS": "Samoa",
+    "SM": "San Marino", "ST": "Sao Tome and Principe",
+    "SA": "Saudi Arabia", "SN": "Senegal", "RS": "Serbia",
+    "SC": "Seychelles", "SL": "Sierra Leone", "SG": "Singapore",
+    "SK": "Slovakia", "SI": "Slovenia", "SB": "Solomon Islands",
+    "SO": "Somalia", "ZA": "South Africa", "KR": "South Korea",
+    "SS": "South Sudan", "ES": "Spain", "LK": "Sri Lanka",
+    "VC": "Saint Vincent and the Grenadines", "PS": "Palestine",
+    "SD": "Sudan", "SR": "Suriname", "SZ": "Swaziland", "SE": "Sweden",
+    "CH": "Switzerland", "SY": "Syria", "TW": "Taiwan",
+    "TJ": "Tajikistan", "TZ": "Tanzania", "MK": "Macedonia",
+    "TH": "Thailand", "TG": "Togo", "TO": "Tonga",
+    "TT": "Trinidad and Tobago", "TN": "Tunisia", "TR": "Turkey",
+    "TM": "Turkmenistan", "TV": "Tuvalu", "GB": "United Kingdom",
+    "US": "United States", "UG": "Uganda",
+    "UA": "Ukraine", "AE": "United Arab Emirates", "UY": "Uruguay",
+    "UZ": "Uzbekistan", "VU": "Vanuatu", "VE": "Venezuela",
+    "VN": "Vietnam", "YE": "Yemen", "ZM": "Zambia", "ZW": "Zimbabwe"
+}
+
+
+def get_mautic_client():
+    if settings.FEATURES.get("MAUTIC_ENABLED"):
+        return MauticBasicAuthClient(
+            settings.MAUTIC_BASE,
+            username='edraakapi',
+            password='2SgGiNCsYREROBzwHQ/aoe8qGT7RygU0UPbEspKJRtM='
```

Comment:
You forgot to read the credentials from the settings

Reply:
Oops